### PR TITLE
Implementing multiple docs navigation

### PR DESCRIPTION
**Pull request by thinkholic**

### CODE REVIEW

File: src/docs/index.js [hunk 1]
```diff
@@ -1,75 +1,65 @@
...
+  if (!docId) {
+    docId = getDocs[0].id;
+  }
+
+  return data[docId].sections;
 }
 
-export function getItem(sectionId, itemId) {
-  const section = data.sections.find(section => section.id === sectionId);
+export function getItem(docId, sectionId, itemId) {
```
Comment (arunoda) [inline src/docs/index.js]: Change docId to `category`.
Change that in all other places as well.

**Comment by thinkholic** [comment]
@arunoda Updated

**Comment by arunoda** [comment]
@thinkholic storybook is failing because of these changes.
Could you fix that?

**Comment by thinkholic** [comment]
@arunoda Fixed

**Comment by arunoda** [comment]
@thinkholic we need to do something as well.
Check [this](https://github.com/kadirahq/getstorybook.io/blob/master/src/components/Docs/index.js#L11).

Here we keep the "React Storybook Docs" as fixed. But since now we have more docs.
It's pretty great if we can set it dynamically.

**Comment by thinkholic** [comment]
Done.

**Pull request merged**